src/submission/submission-entities.ts

```diff
@@ -273,12 +273,20 @@ export const BatchNameRegex: Record<ClinicalEntityType, RegExp[]> = {
   [ClinicalEntityType.REGISTRATION]: [/^sample_registration.*\.tsv$/],
   [ClinicalEntityType.DONOR]: [/^donor.*\.tsv$/],
   [ClinicalEntityType.SPECIMEN]: [/^specimen.*\.tsv$/],
-  [ClinicalEntityType.PRIMARY_DIAGNOSES]: [/^primary_diagnosis.*\.tsv/],
+  [ClinicalEntityType.PRIMARY_DIAGNOSIS]: [/^primary_diagnosis.*\.tsv/],
 };
 
 // assumption: one field uniquely identifies a clinical type record in a batch of records
 export const ClinicalUniqueIndentifier = {
   [ClinicalEntityType.DONOR]: FieldsEnum.submitter_donor_id,
   [ClinicalEntityType.SPECIMEN]: FieldsEnum.submitter_specimen_id,
-  [ClinicalEntityType.PRIMARY_DIAGNOSES]: FieldsEnum.submitter_donor_id,
+  [ClinicalEntityType.PRIMARY_DIAGNOSIS]: FieldsEnum.submitter_donor_id,
 };
+
+export interface RecordsToSubmitterDonorIdMap {
+  [donoSubmitterId: string]: DonorRecordsOrganizer;
+}
+
+export interface DonorRecordsOrganizer {
```

Comment:
rename `ClinicalSubmissionMap`

Reply:
feel free to argue about my suggestions im just trying to avoid the conflict with the donor clinical model